### PR TITLE
Email designs

### PR DESCRIPTION
**Pull request by ghophp**
New emails designs.. the emails that you see on this Pull Request are generated by a small tool created to generate yetu emails... take a look at the tool to see how it works: https://github.com/yetu/yetu-mails

### CODE REVIEW

**Comment by larsblumberg** [comment]
LGTM

File: conf/routes [hunk 1]
```diff
@@ -90,5 +90,4 @@ GET            /setup/confirmedmailerror            @com.yetu.oauth2provider.con
 GET            /setup/confirmedmail/:mailToken      @com.yetu.oauth2provider.controllers.setup.SetupController.handleSignUp(mailToken)
 
 # Download setup app
-GET            /setup/download                      @com.yetu.oauth2provider.controllers.setup.SetupController.download
-
```
Comment (ElliDy) [inline conf/routes]: Is this just formatting change? I can not see any difference.

Comment (ghophp) [inline conf/routes]: Yeah.. maybe a line or something.. I can checkout routes file from master to avoid.. just a second..

**Comment by ElliDy** [comment]
LGTM

**Pull request merged**